eos/utils/szego-polynomial_TEST.cc

```diff
@@ -37,45 +37,56 @@ class SzegoPolynomialTest :
-				}
+                    TEST_CHECK_RELATIVE_ERROR(p0, +0.6351351032391984, 1.0e-5);
+                    TEST_CHECK_RELATIVE_ERROR(p1, -0.847745,           1.0e-5);
+                    TEST_CHECK_RELATIVE_ERROR(p2, +1.54489,            1.0e-5);
+                    TEST_CHECK_RELATIVE_ERROR(p3, -2.82388,            1.0e-5);
+                    TEST_CHECK_RELATIVE_ERROR(p4, +5.166081,           1.0e-5);
+                    TEST_CHECK_RELATIVE_ERROR(p5, -9.464102,           1.0e-5);
+                }
 
-				{
-					const auto [p0, p1, p2, p3, p4, p5] = p(+0.0);
+                {
+                    const auto [p0, p1, p2, p3, p4, p5] = p(+0.0);
 
-					TEST_CHECK_RELATIVE_ERROR(p0, +0.6351351032391984, 1.0e-5);
-					TEST_CHECK_RELATIVE_ERROR(p1, -0.749503,           1.0e-5);
-					TEST_CHECK_RELATIVE_ERROR(p2, +1.304458,           1.0e-5);
-					TEST_CHECK_RELATIVE_ERROR(p3, -2.237009,           1.0e-5);
-					TEST_CHECK_RELATIVE_ERROR(p4, +3.834085,           1.0e-5);
-					TEST_CHECK_RELATIVE_ERROR(p5, -6.577731,           1.0e-5);
-				}
+                    TEST_CHECK_RELATIVE_ERROR(p0, +0.6351351032391984, 1.0e-5);
+                    TEST_CHECK_RELATIVE_ERROR(p1, -0.749503,           1.0e-5);
+                    TEST_CHECK_RELATIVE_ERROR(p2, +1.304458,           1.0e-5);
+                    TEST_CHECK_RELATIVE_ERROR(p3, -2.237009,           1.0e-5);
+                    TEST_CHECK_RELATIVE_ERROR(p4, +3.834085,           1.0e-5);
+                    TEST_CHECK_RELATIVE_ERROR(p5, -6.577731,           1.0e-5);
+                }
 
-				{
-					const auto [p0, p1, p2, p3, p4, p5] = p(+0.1);
+                {
+                    const auto [p0, p1, p2, p3, p4, p5] = p(+0.1);
 
-					TEST_CHECK_RELATIVE_ERROR(p0, +0.6351351032391984, 1.0e-5);
-					TEST_CHECK_RELATIVE_ERROR(p1, -0.651261,           1.0e-5);
-					TEST_CHECK_RELATIVE_ERROR(p2, +1.09668,            1.0e-5);
-					TEST_CHECK_RELATIVE_ERROR(p3, -1.76188,            1.0e-5);
-					TEST_CHECK_RELATIVE_ERROR(p4, +2.82970,            1.0e-5);
-					TEST_CHECK_RELATIVE_ERROR(p5, -4.54691,            1.0e-5);
-				}
-			}
+                    TEST_CHECK_RELATIVE_ERROR(p0, +0.6351351032391984, 1.0e-5);
+                    TEST_CHECK_RELATIVE_ERROR(p1, -0.651261,           1.0e-5);
+                    TEST_CHECK_RELATIVE_ERROR(p2, +1.09668,            1.0e-5);
+                    TEST_CHECK_RELATIVE_ERROR(p3, -1.76188,            1.0e-5);
+                    TEST_CHECK_RELATIVE_ERROR(p4, +2.82970,            1.0e-5);
+                    TEST_CHECK_RELATIVE_ERROR(p5, -4.54691,            1.0e-5);
+                }
+
+                {
+                    const auto [p0, p1, p2, p3, p4, p5] = p(complex<double>(0.4, 0.916515139));
+
+                    TEST_CHECK_RELATIVE_ERROR(p0, +0.6351351032391984, 1.0e-5);
+                    TEST_CHECK_RELATIVE_ERROR_C(p1, complex<double>(-0.35653430,  0.90040363),  1.0e-5);
```

Comment:
Has this been checked against your notebook?

Reply:
Sure :)